docs/quickstarts/python-quickstart.md

```diff
@@ -0,0 +1,203 @@
+  ```
+
+
+## Run a gRPC application
+
+From the `examples/python/helloworld` directory:
+
+1. Run the server
+
+   ```sh
+   $ python2.7 greeter_server.py &
+   ```
+
+2. In another terminal, run the client
+
+   ```sh
+   $ python2.7 greeter_client.py
+   ```
+
+Congratulations! You've just run a client-server application with gRPC.
+
+## Update a gRPC service
+
+Now let's look at how to update the application with an extra method on the server for the client to call. Our gRPC service is defined using protocol buffers: you can find out lots more about how this works in [What is gRPC?]() and [gRPC Basics: Python][]. For now all you need to know is that both the server and the client "stub" have a `SayHello` RPC method that takes a `HelloRequest` parameter from the client and returns a `HelloResponse` from the server, and that this method is defined like this:
+
+```
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+Let's update this so that the `Greeter` service has two methods. In your IDE or text editor [what should we recommend, Python people?], open `examples/proto/helloworld.proto` and update it with a new `SayHelloAgain` method, with the same request and response types:
+
+```
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+  // Sends another greeting
+  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+Don't forget to save the file!
+
+## Generate gRPC code
+
+Next we need to update the gRPC code used by our application to use the new service definition. From the `examples/python/helloworld` directory:
+
+```
+$ ./run_codegen.sh
+```
+
+This script invokes the protocol buffer compiler as:
+
+```
+$ protoc -I ../../protos --python_out=. --grpc_out=. --plugin=protoc-gen-grpc=&#96;which grpc&lowbar;python&lowbar;plugin&#96; ../../protos/helloworld.proto</pre>
+```
+
+This generates `helloworld_pb2.py`, which contains our generated client and server classes, as well as classes for populating, serializing, and retrieving our request and response types.
+
+## Update and run the application
+
+We now have new generated server and client code, but we still need to implement and call the new method in the human-written parts of our example application.
+
+### Update the server
+
+In the same directory, open `greeter_server.py`. Implement the new method like this:
+
+```
+class Greeter(helloworld_pb2.BetaGreeterServicer):
```

Comment:
Drop "Beta" from this name, [migrating this part of this example from the Beta API to the GA API](https://github.com/grpc/grpc/issues/4977).